android/src/main/java/com/balthazargronon/RCTZeroconf/ZeroconfModule.java

```diff
@@ -115,32 +79,91 @@ protected void sendEvent(ReactContext reactContext,
                 .emit(eventName, params);
     }
 
-    private class ZeroResolveListener implements NsdManager.ResolveListener {
+    private String getServiceName(String fqdn) {
```

Comment:
fqdn it something like ServiceName._protocol._local. Do you think we can use it as service name and let user to decide what to do?